quiz/quiz.go

```diff
@@ -0,0 +1,182 @@
+// This package implements the quiz, with a variation on part 2.  
+// Instead of having an timer for the entire test, it handles a timer 
+// for each question.  This was more challenging, since characters 
+// entered for previous questions had to be properly discarded
+// to get the desired behavior.
+
+package main
+
+import (
+	"bufio"
+	"encoding/csv"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+type QuizItem struct {
```

Comment:
All exported elements should have comments. You have to enable this linting in the goland options menu.

Reply:
Thanks.  I've enabled linting in Visual Studio code now.  The goland IDE is still crashing on my machine